order/tradingfees.go

```diff
@@ -219,23 +219,23 @@ func (t *AccountTally) CalcMakerDelta(feeSchedule terms.FeeSchedule,
 // order where the account is involved on the taker side. It returns the
 // balance delta, taker fees paid, and execution fee paid.
 func takerDelta(feeSchedule terms.FeeSchedule,
-	price FixedRatePremium, totalSats, selfChanBalance btcutil.Amount,
+	price FixedRatePremium, premiumAmt, takerAmt btcutil.Amount,
 	duration uint32) (btcutil.Amount, btcutil.Amount, btcutil.Amount) {
 
 	// Calculate the premium based on the duration the capital will be
 	// locked for.
-	satsPremium := price.LumpSumPremium(totalSats, duration)
+	satsPremium := price.LumpSumPremium(premiumAmt, duration)
```

Comment:
I think the distinction of channel amt vs premium amount here (and elsewhere in the PR is a bit confusing). For example, in this line, we're taking the premium and computing another premium...

Reply:
+100, naming will be comprehensively defined in a new commit

Reply:
From the "outside" functions it makes sense to call it "premiumAmt" as "the amount that we take into account to calculate the premium of this match" however I agree it is confusing in the last step, when we calculate the premium that we have to pay for this match. I changed it for `baseAmt` in the last step, not sure what better name could fit here :disappointed: